package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@evolv/javascript-sdk",
+  "version": "1.0.0",
+  "description": "This is the official JavaScript client for the Evolv Technologies optimization platform.",
+  "main": "src/index.js",
+  "type": "module",
+  "homepage": "https://www.evolv.ai",
+  "bugs": {
+    "url": "https://github.com/evolv-ai/javascript-sdk/issues",
+    "email": "support@evolv.ai"
+  },
+  "directories": {
+    "doc": "./docs"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/evolv-ai/javascript-sdk"
+  },
+  "engines": {
+    "node": ">=13"
```

Comment:
Why the limitation to Node 13? Node 13 isn't even at LTS yet (or rather 14 hasn't been released yet). That's going to bother some enterprise users, isn't it? It also rules out using this library in AWS Lambdas.

If the reason is to allow for ES import/exports, then I have to advocate for transpiling again.